open-source-trumps-innersource.md

```diff
@@ -0,0 +1,40 @@
+## Title
+Open Source trumps InnerSource
+
+## Statement of Problem
+* People find the InnerSource project but, after all things are considered, even if the InnerSource component meets their needs, they still go with the open source component.
+
+## Context
+* People look for software through external search engines.
+* Different Business Lines (BLs) are selecting different SW components to do the same function, resulting in inconsistencies in the user experience.
+
+## Forces 
+* There is a perception that the open source components are higher quality and more reusable. The channels to get any needed changes are more obvious with open source than with internal components. Can easily grab the open source whereas internally developed component has less visibility and ease of availability. (This could be separate Pattern). All things equal, they gravitate towards open source.
+* Open source should be voluntary; mandating people to use internally developed software goes against open source principles.
+* Internally developed components should have advantages over competing open source. If you can clearly demonstrate this, it can be persuasive.
+* Part of the reason this is a problem: if different teams are picking different open source components, the situation makes interoperability and integration more difficult; and this could potentially impact the user experience (inconsistency in behaviors).
+* Depending on the regulations within a company, it might be easier to use a corporate licensed, InnerSource component than an open source component which is licensed in a way that requires the company to disclose IP.
+* There may be many silos in the company; it would then be difficult to reach the developer base across those silos.
+* It can be hard to find stuff in github (especially if names are cryptic and keywords aren't used). See [Poor Naming Conventions](https://github.com/paypal/InnerSourcePatterns/pull/59).
+
+## Sketch
+See figure 1 in https://drive.google.com/open?id=0B7_9iQb93uBQNlJ0YU5wWmpWYUU 
+
+## Resolution
+* Corporate requirement that internally developed components should be evaluated before we go outside to search for open source component.
```

Comment:
I doubt the ability for such a requirement to have real, lasting, and actual affect on employee behavior.

Reply:
It may depend on the corporate culture or how this requirement is put in place (there might be a process for finding and evaluating software components or for using open source software). I think it is doable to require this step (we use it in our company).

Reply:
If this is the core solution of this pattern, then I propose we add language to the context which makes it clear that this solution could be successfully applied.

Reply:
Context contains:
> * The company culture is based on strong governance.

So I think this thread is addressed. Resolving.